### PR TITLE
Create utils.file module for file-related operations

### PR DESCRIPTION
**Pull request by tatiana**
Relates to: #98  (part 2)

### CODE REVIEW

File: src/astro/utils/file.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+
+def get_size(filepath):
+    """
+    Return the size (bytes) of the given file.
+
+    :param filepath: Path to a file in the filesystem
+    :type filepath: str
+    :return: File size in bytes
+    :rtype: int
```
Comment (dimberman) [inline src/astro/utils/file.py:14]: Huge +1 for starting to add docstrings :)

**Review by dimberman — APPROVED** [review]

File: src/astro/utils/file.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    Return a FileType given the filepath. Uses a naive strategy, using the file extension.
+
+    :param filepath: URI or Path to a file
+    :type filepath: str
+    :return: The filetype (e.g. csv, ndjson, json, parquet)
+    :rtype: astro.constants.FileType
+    """
+    extension = filepath.split(".")[-1]
+    try:
+        filetype = getattr(FileType, extension.upper())
```
Comment (utkarsharma2) [inline src/astro/utils/file.py:31]: Can we do something like - 
filetype = FileType(extension.upper())

File: src/astro/utils/file.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+def is_binary(filetype):
+    """
+    Return a FileType given the filepath. Uses a naive strategy, using the file extension.
+
+    :param filetype: File type
+    :type filetype: astro.constants.FileType
+    :return: True or False
+    :rtype: bool
+    """
+    if filetype == FileType.PARQUET:
```
Comment (utkarsharma2) [inline src/astro/utils/file.py:46]: This can be simplified to -
return filetype == FileType.PARQUET

File: src/astro/utils/file.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    """
+    Checks if a file is small enough to be loaded into a Pandas dataframe in memory efficently.
+    This value was obtained through performance tests.
+
+    :param filepath: Path to a file in the filesystem
+    :type filepath: str
+    :return: If the file is small enough
+    :rtype: boolean
+    """
+    size_in_bytes = get_size(filepath)
```
Comment (utkarsharma2) [inline src/astro/utils/file.py:61]: Why do we need extra variable here? 
can we do something like - 
return get_size(filepath) <= LOAD_DATAFRAME_BYTES_LIMIT

File: tests/utils/test_file.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+sample_filetypes = [items[0] for items in sample_filepaths_per_filetype]
+sample_filepaths = [items[1] for items in sample_filepaths_per_filetype]
+sample_filepaths_ids = [items[0].value for items in sample_filepaths_per_filetype]
+
+
+def test_get_size():
+    assert get_size(sample_file) == 65
+
+
+def describe_get_filetype():
```
Comment (utkarsharma2) [inline tests/utils/test_file.py:25]: I like this style for grouping pytest!

**Pull request merged**